mmv1/third_party/terraform/services/vmwareengine/data_source_google_vmwareengine_private_cloud.go

```diff
@@ -13,8 +12,6 @@ func DataSourceVmwareenginePrivateCloud() *schema.Resource {
 
 	dsSchema := tpgresource.DatasourceSchemaFromResourceSchema(ResourceVmwareenginePrivateCloud().Schema)
 	tpgresource.AddRequiredFieldsToSchema(dsSchema, "name", "location")
-	tpgresource.AddOptionalFieldsToSchema(dsSchema, "project")
```

Comment:
Could you explain this change? I'm wondering if this could be a breaking change, if we are saying that `project` was an optional parameter previously, but now it is required, or throws some sort of error if present.

Reply:
My bad, lets keep it optional for now. Thanks for pointing it out!

Reply:
Sounds good!